src/plugins/gauge/GaugeCompositionPolicy.js

```diff
@@ -0,0 +1,27 @@
+export default function GaugeCompositionPolicy(openmct) {
+    function hasNumericTelemetry(domainObject) {
+        const hasTelemetry = openmct.telemetry.isTelemetryObject(domainObject);
+        if (!hasTelemetry) {
+            return false;
```

Comment:
We should add to be sure the vue component renders properly when the telemetry is unavailable.